config-overrides.js

```diff
@@ -1,9 +1,18 @@
-module.exports = function override(webpackConfig) {
-  webpackConfig.module.rules.push({
-    test: /\.mjs$/,
-    include: /node_modules/,
-    type: "javascript/auto",
-  });
+module.exports = {
+  webpack: function override(webpackConfig) {
+    webpackConfig.module.rules.push({
+      test: /\.mjs$/,
+      include: /node_modules/,
+      type: "javascript/auto",
+    })
 
-  return webpackConfig;
-};
+    return webpackConfig
+  },
+  devServer: function (configDevServer) {
+    return function (proxy, allowedHosts) {
+      const config = configDevServer(proxy, allowedHosts)
+      return config
+      // return { ...config, public: "wallet.end-labs.local" }
```

Comment:
Comment out L14 and uncomment L15 to serve the app from a different domain name during local development. Make sure to add that domain in your `/etc/hosts` as well (e.g. `127.0.0.1  wallet.end-labs.local`).